### DISCUSSION TITLE
Can I setup/select download folder?

### DISCUSSION THREAD

**Discussion by sevencan** [Lofter1/anyflip-downloader#28] [General]
As far as I can see, the files end up in system32 folder. Can I select the download folder?

**Comment by KatnipSniffer** [comment]
In Powershell, before running `anyflip-downloader`, run `set-location *directory*`
This tells Powershell where you want to run whatever you want to run. 
For example `set-location C:\Program Files\Anyflip Downloads`
I had to go through too many steps from two different guides, downloading go tools (which wasn't necessary at all for this) only to figure out that I hadn't set any directory so Powershell defaulted to my user file. My first time using Powershell for anything lol

**Reply by astreopoli** [reply]
Can you help with anyflip-downloader, I have tried downloading using anyflip-downloader but I can't find the download results in the form of a .pdf file, where are the download results located? Can you help with an example like >>> anyflip-downloader -temp-download-folder <temp folder name> <url to book> ? For example, if I save it on (drive: C), how do I write the script via Power Shell? is it like this >>> anyflip-downloader -temp-download-folder C:\Users\account\Documents https://anyflip.com/zlumq/uzjy

**Reply by astreopoli** [reply]
PS C:\Windows\system32> . { iwr -useb https://raw.githubusercontent.com/Lofter1/anyflip-downloader/main/scripts/install.ps1 } | iex;
Detected architecture amd64
x LICENSE
x README.md
x anyflip-downloader.exe
anyflip-downloader has been installed to C:\Users\akunp\AppData\Local\anyflip-downloader.
PS C:\Windows\system32> anyflip-downloader set-location C:\Program Files\Anyflip Downloads https://anyflip.com/zlumq/uzjy
Preparing to download
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
main.sanitizeURL(0xc00013cb40)
        /home/runner/work/anyflip-downloader/anyflip-downloader/main.go:116 +0xf4
main.prepareDownload(0xc00013cb40)
        /home/runner/work/anyflip-downloader/anyflip-downloader/main.go:78 +0x46
main.main()
        /home/runner/work/anyflip-downloader/anyflip-downloader/main.go:52 +0x1c5
PS C:\Windows\system32>

==========================================
**Is this an example of what I did? but there is an error, please help, so you can find the download results**

**Reply by KatnipSniffer** [reply]
Run `set-location C:\Users\*account*\Documents`,
Run `anyflip-downloader` 
(These are the whole commands, no addional input required.)

If no issue occurs during the download, the pdf should be downloaded to your documents. 
If you've already ran `anyflip-downloader` before setting a location, check your user file (`C:\Users\*account*`), the pdf should be there.

This is how it works for me, and sadly, I cannot provide more complex information as I am not proficient with this digital tool.
